### ISSUE TITLE
Options Error - All fields required

### ISSUE THREAD

**Issue by franzbascope** [skynet2/ngx-google-places-autocomplete#97]
When I try to input options it is showing me this error:

Type '{ componentRestrictions: { country: string; }; }' is missing the following properties from type 'Options': bounds, types, fields, strictBounds, origin

17         [options]="{
            ~~~~~~~

The error is because in the Options Interface all properties are required. However, I think these should be optional. For example,
I want to restrict the country , but I do not want to set bounds, and this is not possible right now, please help. Thank you

  Versions
"@angular/core": "^12.0.0",
  "ngx-google-places-autocomplete": "^2.0.5",

**Comment by finitekaren** [comment]
you can set to null but i agree that these should be optional. 

google docs have these as optional https://developers.google.com/maps/documentation/javascript/reference/places-widget#AutocompleteOptions

**Comment by mayurgudi** [comment]
> you can set to null but i agree that these should be optional.
> 
> google docs have these as optional https://developers.google.com/maps/documentation/javascript/reference/places-widget#AutocompleteOptions

No, it shows all fields must be entered and they can't be set to null or undefined. Expected data type is shown. This requirement made me downgrade version of this but then "options" is only getting treated as unknown property !

**Comment by mayurgudi** [comment]
HTML
<input type="text" ngx-google-places-autocomplete [options]="myOptions">

TS
import { Options } from 'ngx-google-places-autocomplete/objects/options/options';

myOptions: any;         // class variable

constructor() {
    this.myOptions = new Options({ componentRestrictions: { country: 'code' } });   // can use any of the restriction(s)
}
